pkg/daemon/daemon.go

```diff
@@ -580,8 +590,17 @@ func (dn *Daemon) completeDrain() error {
 		return err
 	}
 
+	if utils.ClusterType == utils.ClusterTypeOpenshift {
+		glog.Infof("completeDrain(): resume MCP %s", dn.mcpName)
+		pausePatch := []byte("{\"spec\":{\"paused\":false}}")
+		if _, err := dn.mcClient.MachineconfigurationV1().MachineConfigPools().Patch(context.Background(), dn.mcpName, types.MergePatchType, pausePatch, metav1.PatchOptions{}); err != nil {
```

Comment:
After rebooting the node, dn.mcpName is empty which results in failure updating the api.
dn.mcpName is assigned in drainNode function, but drainNode is not always called after rebooting.

Reply:
The `dn.mcpName` is assigned in `getNodeMachinePool`, which will be invoked in `nodeStateSyncHandler`. So it will always be assigned before we consume it.

Reply:
I'm testing in a baremetal environment, what I observed was:
```
E0401 01:01:52.621664    5940 daemon.go:592] completeDrain(): failed to resume MCP : resource name may not be empty
E0401 01:01:52.621674    5940 daemon.go:564] nodeStateSyncHandler(): failed to complete draining: resource name may not be empty
```
The mcp never comes back.

Reply:
ignore this, I might use wrong build.